packages/cli/lib/lib/webpack/webpack-client-config.js

```diff
@@ -183,9 +185,41 @@ function isProd(config) {
 		prodConfig.plugins.push(
 			new BabelEsmPlugin({
 				filename: '[name].[chunkhash:5].esm.js',
-				chunkFilename: '[name].chunk.[chunkhash:5].esm.js'
+				chunkFilename: '[name].chunk.[chunkhash:5].esm.js',
```

Comment:
`.mjs`?

Reply:
Webpack has issues with mjs IIRC

Reply:
https://github.com/webpack/webpack/issues/5686
https://github.com/webpack/webpack/pull/4691

Works fine in Webpack 4 :)